app/components/HouseholderForm/HouseholderIdentityForm.tsx

```diff
@@ -0,0 +1,188 @@
+  firstName: HouseholderIdentitySchema.firstName(),
+  lastName: HouseholderIdentitySchema.lastName(),
+  fatherName: HouseholderIdentitySchema.fatherName(),
+  nationalId: HouseholderIdentitySchema.nationalId(),
+  ssn: HouseholderIdentitySchema.ssn(),
+});
+
+const useStyles = createStyles(theme => ({
+  textInput: {
+    label: {
+      color: theme.colors.fgSubtle[6],
+    },
+  },
+  dateInput: {
+    label: {
+      color: theme.colors.fgSubtle[6],
+    },
+  },
+}));
+
+export const HouseholderIdentityForm = () => {
+  const t = messages.householder.householderIdentityForm;
+  const { classes } = useStyles();
+  const { register, formState } = useForm<FormSchema>({
+    resolver,
+    mode: 'onChange',
+  });
+
+  return (
+    <form>
+      <Stack spacing={25}>
+        <Group position="apart">
+          <Title order={4} color="fgMuted" weight="bold">
+            {t.title}
+          </Title>
+          <Button
+            {...createTestAttr(Ids.submitBtn)}
+            type="submit"
+            size="sm"
+            leftIcon={<CheckMark />}
+          >
+            {t.submitBtn}
+          </Button>
+        </Group>
+        <SimpleGrid cols={3} spacing="lg" verticalSpacing={20}>
+          <TextInput
+            wrapperProps={createTestAttr(Ids.firstNameInput)}
+            {...register('firstName')}
+            className={classes.textInput}
+            label={`${t.nameInput.label}:`}
+            placeholder={t.nameInput.placeholder}
+            error={formState.errors.firstName?.message}
+          />
+          <TextInput
+            wrapperProps={createTestAttr(Ids.lastNameInput)}
+            {...register('lastName')}
+            className={classes.textInput}
+            label={`${t.lastNameInput.label}:`}
+            error={formState.errors.lastName?.message}
+            placeholder={t.lastNameInput.placeholder}
+          />
+          <TextInput
+            wrapperProps={createTestAttr(Ids.fatherNameInput)}
+            {...register('fatherName')}
+            className={classes.textInput}
+            label={`${t.fatherNameInput.label}:`}
+            placeholder={t.fatherNameInput.placeholder}
+            error={formState.errors.fatherName?.message}
+          />
+          <Select
+            wrapperProps={createTestAttr(Ids.nationalityInput)}
+            data={[
+              {
+                value: t.nationalityInput.data.value,
+                label: t.nationalityInput.data.label,
+              },
+            ]}
+            placeholder={t.nationalityInput.placeholder}
+            label={`${t.nationalityInput.label}:`}
+          />
+          <TextInput
+            wrapperProps={createTestAttr(Ids.nationalIdInput)}
+            error={formState.errors.nationalId?.message}
+            className={classes.textInput}
+            {...register('nationalId')}
+            placeholder={t.nationalIdInput.placeholder}
+            label={`${t.nationalIdInput.label}:`}
```

Comment:
```suggestion
            label={t.nationalIdInput.label}
```

why won't we just put the `:` inside the messages itself?

Reply:
Idk. is it a better idea?
@ASafaeirad What's ur opinion?